src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubOptions.cs

```diff
@@ -368,10 +373,21 @@ public string Format()
                 };
             }
 
+            JObject partitionManagerOptions = null;
+            if (partitionManagerOptions != null)
+            {
+                eventProcessorOptions = new JObject
+                {
+                    { nameof(PartitionManagerOptions.LeaseDuration), EventProcessorOptions.EnableReceiverRuntimeMetric },
+                    { nameof(PartitionManagerOptions.RenewInterval), EventProcessorOptions.InvokeProcessorAfterReceiveTimeout },
```

Comment:
These values look wrong? Copy paste error? Need to update the test so it catches this.